sim/make/config.mk

```diff
@@ -1,28 +1,15 @@
-# FireSim Target Agnostic Make Fragment
-#
-# Defines make targets for:
-# - invoking Golden Gate (phony: verilog / compile)
-# - building a simulation driver (phony: f1)
-# - populating an FPGA build directory (phony: replace-rtl)
-# - generating new runtime configurations (phony: conf)
-# - compiling meta-simulators (phony: verilator, vcs, verilator-debug, vcs-debug)
-#
+# See LICENSE for license details.
+
+##########################
+# Target Configuration   #
+##########################
 
 # The prefix used for all Golden Gate-generated files
-BASE_FILE_NAME ?=
+BASE_FILE_NAME ?= FireSim-generated
```

Comment:
I'd defer to you but you might as well lock this down.

Reply:
Perfect.